Controllers/controller.py

```diff
@@ -106,3 +109,20 @@ def open_file_dialog(self):
             url = file_dialog.selectedUrls()[0]
             self._media_player.setSource(url)
             self._media_player.play()
+            
+    @Slot()
+    def play_video(self):
```

Comment:
Add a docstring to this play_video function. Use the same style as the docstrings of other functions.